### PR TITLE
Clean markdown files

### PR DESCRIPTION
**Pull request by st-Wook**
Updated

- Add black line
- Remove trailing whitespace
- Emphasis style consistently

### CODE REVIEW

**Review by tristanbob — APPROVED** [review]
@st-Wook thanks so much for helping us improve our repo!  
We really appreciate people with attention to detail.  :)

File: README.md [hunk 1]
```diff
@@ -21,15 +21,18 @@ If you've created an example with GDevelop, you can submit it to be shared with
 
 1. **Create your game** with GDevelop.
 2. Make sure to follow the [requirements and best practices on this page](http://wiki.compilgames.net/doku.php/gdevelop5/community/guide-for-submitting-an-example).
-  > Note that for now, we're trying to keep a fairly high quality bar for examples, so you might be asked to adapt your game according to reviewer feedbacks. Don't feel bad about this! This is normal process and here to help making examples as good as possible for new users. If we take too much time to review your example, you can send a ping on the issue. ⏰
+  > Note that for now, we're trying to keep a fairly high quality bar for examples, so you might be asked to adapt your game according to reviewer feedbacks.
+  > Don't feel bad about this! This is normal process and here to help making examples as good as possible for new users.
+  > If we take too much time to review your example, you can send a ping on the issue. ⏰
```
Comment (tristanbob) [inline README.md:26]: I just noticed that this change didn't result in a visible change. 
What was your goal with this change?  Were you trying to make a bulleted list?

Comment (st-Wook) [reply README.md:26]: Changed to make it easy to understand the content when reading as a file!

Comment (tristanbob) [reply README.md:26]: That makes sense, thanks for explaining!  :)

**Pull request merged**